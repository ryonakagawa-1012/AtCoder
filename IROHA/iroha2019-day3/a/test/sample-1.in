25 10 800 250 6 4 15 7 5 5 17 31 334 777 616 88 919 331 270 27 555 555 555 1000 100 10
