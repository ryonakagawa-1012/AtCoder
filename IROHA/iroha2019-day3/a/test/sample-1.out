15
1050
0
10
aaaji
6
1201670
720
